configs/nextflow.hopper.config

```diff
@@ -335,6 +301,50 @@ modycf {
     params.gens_accessdir = "/access/wgs/plot_data"
     params.accessdir = "/access/modycf/"
     params.verifybamid2_svdprefix = "/fs1/resources/ref/hg38/VerifyBamID/resource/exome/no_chr/1000g.phase3.10k.b38.exome.vcf.gz.dat"
+
+    // trio rank models not used in mody:
+    params.rank_model = "/fs1/resources/scout/rank_models_38/rank_model_cmd_v4__VEP111.ini"
+    params.svrank_model = "/fs1/resources/scout/rank_models_38/svrank_model_v1.0__VEP111.ini"
+  }
+
+exome {
```

Comment:
Could we remove this one completely? So we don't need to maintain / keep track of it